rllib/algorithms/impala/tests/test_impala_learner.py

```diff
@@ -76,6 +76,9 @@ def test_impala_loss(self):
                 _enable_rl_module_api=True,
             )
         )
+        # We have to set exploration_config here manually because setting it through
```

Comment:
TODO (Artur): See if setting Impala's default to {} pose any issues. Deprecate the current default and set it to {}.

Reply:
Use better wording :)